torch_geometric/utils/to_dgnn.py

```diff
@@ -0,0 +1,31 @@
+import scipy.sparse as sp
+import torch
+
+
+def to_dgnn(edge_index):
+    device = edge_index.device
+
+    col, row = edge_index.cpu()
+    numlist = torch.arange(col.size(0), dtype=torch.int32)
+    num_nodes = torch.max(col) - torch.min(col) + 1
+
+    adj_csr = sp.csr_matrix((numlist.numpy(), (row, col)),
```

Comment:
Can we implement this via PyTorch manually? Current implementation is CPU-only :(